tensorflow/core/api_def/base_api/api_def_Greater.pbtxt

```diff
@@ -4,5 +4,15 @@ op {
   description: <<END
 *NOTE*: `Greater` supports broadcasting. More about broadcasting
 [here](http://docs.scipy.org/doc/numpy/user/basics.broadcasting.html)
+
+```python
```

Comment:
Can you add a line saying `Example:` before all these examples?

Reply:
And you need an empty line before the `Example:` otherwise it won't be recognized as a section heading.